src/elements/content-preview/ContentPreview.js

```diff
@@ -789,6 +808,7 @@ class ContentPreview extends React.PureComponent<Props, State> {
         }
 
         const previewOptions = {
+            advancedContentInsights,
```

Comment:
Sure @tjuanitas . Preview will use the enabled, isActive, ownerEId and userId. Since preview needs to know if AdvancedContentInsights is enabled (but is not using the onChange method), I'll prefer to keep the enabled prop.

Reply:
what about if `enabled` is false, then the parent component doesn't pass anything for the `advancedContentInsights` prop?

```
// in some parent component in the webapp

return (
  <ContentPreview
    advancedContentInsights={isFeatureFlipEnabled ? propsForPreview : undefined}
  />
);
```

my concern with `enabled` is that these lower components shouldn't need to know the state of a feature flip. after the feature has been released, how should these components behave? will we always need to pass `enabled` even if the feature is released and then removed?